runner.ts

```diff
@@ -117,11 +121,14 @@ export async function run(source : string, config: Config) : Promise<[Value, Glo
     ${config.functions}
     ${compiled.functions}
     (func (export "exported_func") ${returnType}
+      (local $i i32)
+      (local $length i32)
```

Comment:
I'm confused about these hardcoded variables – what are they for? Wouldn't they also be needed in every function definition? Why can't new temporaries introduced in `lower.ts` do the work of these?

Reply:
We have now remove these two lines. It is used for the **print_str**. Now the new print is encapsulated as an external function.